common/source/docs/common-mamba405-mk2.rst

```diff
@@ -0,0 +1,112 @@
+================================
+Mamba F405 MK2 Flight Controller
+================================
+
+.. image:: ../../../images/mamba_f405mk2_board.jpg
+     :target: ../_images/mamba_f405mk2_board.jpg
+
+https://www.diatone.us/products/mamba-f405-flight-controller-mk2
+
+The Mamba F405 MK2 is a flight controller produced by
+`Diatone <https://www.diatone.us>`__.
+
+.. note::
+
+   Support for this board is available with Plane 4.1.0, and higher
```

Comment:
remove note please....trying to avoid any revs references if possible....created Upcoming features section for things not in stable but in master, however, the boards have been an exception and  we have just put it in since there will not be firmware downloadable except for appropriate revs (in this case latest)

Reply:
done